source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -60,7 +60,7 @@ OpenGLSurface: abstract class extends GpuCanvas {
 		f := func {
 			this context meshShader add("texture0", image)
 			this context meshShader projection = this _projection
-			this context meshShader use()
+			this context meshShader use(null)
```

Comment:
`null` should actually be `this _target` that does not exist in `OpenGLSurface`.